packages/cli/src/commands/migrate.ts

```diff
@@ -232,3 +258,8 @@ function parseModuleList(moduleList: IModule[]): ParsedModuleResult {
   });
   return { edgeList, coreDepList };
 }
+
+// helper function to parse a,b,c to [a, b, c]
+function commaSeparatedArgs(input: string): Array<string> {
```

Comment:
[Seems](https://github.com/rehearsal-js/rehearsal-js/pull/327/files#diff-9c1f10252b43f2fa51543038f9f92b0306f8093823708c92d7d64cb1448ba1dbL10) we prefer to use `string[]`